Consensus-CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Consensus Changelog
+
+The Consensus packages are organized in two bundles: the packages that are
+ledger-agnostic and the packages that are Cardano specific. For each of these
+bundles we keep a separate `CHANGELOG.md` file, and the version number of all
```

Comment:
Maybe also describe here how we achieve a single changelog file for bundles of packages that each live in a different directory: by symlinking the `changelog.d` directories

Reply:
Done